pkg/integrations/manager.go

```diff
@@ -237,15 +238,16 @@ func (m *Manager) instanceConfigForIntegration(cfg Config, i Integration) instan
 	common := cfg.CommonConfig()
 	relabelConfigs := append(m.defaultRelabelConfigs, common.RelabelConfigs...)
 
-	var scrapeConfigs []*config.ScrapeConfig
 	schema := "http"
 	// Check for HTTPS support
-	httpClientConfig := config_util.HTTPClientConfig{}
-	if m.c.SeverUsingTLS {
+	var httpClientConfig = config_util.HTTPClientConfig{}
```

Comment:
Sorry, I meant to do it exactly this way :) 

```suggestion
	var httpClientConfig config_util.HTTPClientConfig
```